deployment/argo-workflows/eks/workflow-planet-scene-id.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: workflow-
+spec:
+  entrypoint: workflow-template
+  parallelism: 30
+  arguments:
+    parameters:
+    - name: activator-input
+      value: |
+        ["S2B_23XNK_20210819_0_L2A"]
+
+  templates:
+  - name: workflow-template
+    dag:
+      tasks:
+      - name: activator
+        template: activator-template
+        arguments:
+          parameters:
+          - name: activator-input
+            value: "{{item}}"
+        withParam: "{{workflow.parameters.activator-input}}"
+
+  - name: activator-template
+    inputs:
+      parameters:
+      - name: activator-input
+    container:
+      image: 513167130603.dkr.ecr.us-east-1.amazonaws.com/activator:latest
+      imagePullPolicy: Always # IfNotPresent
+      env:
+      - name: STAC_API_URI
+        value: "https://franklin.nasa-hsi.azavea.com"
+      - name: GDAL_OUTPUT_FORMAT
+        value: "GTiff"
+      - name: AWS_REQUEST_PAYER
+        value: "requester"
+      - name: GDAL_DISABLE_READDIR_ON_OPEN
+        value: "YES" 
```

Comment:
Could you add the following? (that is the way to set planet key as an env picked from the k8s secrets):

```yml
      - name: PLANET_API_KEY
        valueFrom:
          secretKeyRef:
            name: planet
            key: api-key
```